oauth2/hook.go

```diff
@@ -60,6 +62,7 @@ func RefreshTokenHook(config *config.Provider) AccessRequestHook {
 
 		reqBody := RefreshTokenHookRequest{
 			Subject:         session.GetSubject(),
+			IDTokenExtra:    session.IDTokenClaims().Extra,
```

Comment:
We could include both:

1. session
2. requester

which would give 100% of the context to the hook.